README.md

```diff
@@ -40,7 +40,27 @@ If the MDI file is found and is correct the following actions are taken:
   
 ARGUMENTS
```

Comment:
This one is really weird. On my machine I changed the name a couple of weeks ago already, but apparently this did not change everywhere on GitHub. Do you have any clue what GitHub voodoo spells I need to use to change this?